package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "brave-core-crx-packager",
+  "version": "1.0.0",
+  "description": "Packages component extensions used in the Brave browser",
+  "dependencies": {
+    "ad-block": "brave/ad-block",
+    "tracking-protection": "brave/tracking-protection",
+    "request": "^2.85.0"
+  },
+  "devDependencies": {
+    "commander": "^2.15.1",
+    "crx": "^3.2.1",
+    "mkdirp": "^0.5.1",
+    "replace-in-file": "^3.4.0",
+    "rimraf": "^2.6.2"
```

Comment:
`npm install standard --save-dev`

Then  edit package.json and add a new script for `"lint": "standard",`

Then you can get the normal coding standard we use with `npm run lint`.

You can call `npm run lint -- --fix`  once to most of the problems, all the problems in your case.